db/change_listener.go

```diff
@@ -139,6 +145,15 @@ func (listener *changeListener) Stop() {
 			base.Debugf(base.KeyChanges, "Error closing listener tap feed: %v", err)
 		}
 	}
+
+	// Wait for mutation feed worker to terminate.
+	waitTime := MutationFeedStopMaxWait
+	select {
+	case <-listener.FeedArgs.DoneChan:
+		// Mutation feed worker goroutine is terminated and doneChan is already closed.
+	case <-time.After(waitTime):
+		base.Infof(base.KeyAll, "Timeout after %v of waiting for mutation feed worker to terminate", waitTime)
```

Comment:
Do you agree that this scenario should be unexpected/unusual (listener taking more than 30s to close)?  I think Warnf might be appropriate here.

Reply:
Yeah, I do agree that this scenario should be unexpected/unusual (listener taking more than 30s to close). Added changes to log warning instead of info.